src/parser.rs

```diff
@@ -2272,11 +2276,32 @@ impl<'a> Parser<'a> {
 
     /// Parse an INSERT statement
     pub fn parse_insert(&mut self) -> Result<Statement, ParserError> {
+        let or = if dialect_of!(self is SQLiteDialect) {
+            if self.parse_keywords(&[Keyword::OR, Keyword::REPLACE]) {
+                Some(SqliteOnConflict::Replace)
+            } else if self.parse_keywords(&[Keyword::OR, Keyword::ROLLBACK]) {
+                Some(SqliteOnConflict::Rollback)
+            } else if self.parse_keywords(&[Keyword::OR, Keyword::ABORT]) {
+                Some(SqliteOnConflict::Abort)
+            } else if self.parse_keywords(&[Keyword::OR, Keyword::FAIL]) {
+                Some(SqliteOnConflict::Fail)
+            } else if self.parse_keywords(&[Keyword::OR, Keyword::IGNORE]) {
+                Some(SqliteOnConflict::Ignore)
+            } else if self.parse_keyword(Keyword::REPLACE) {
```

Comment:
I think this is not right? looking at https://sqlite.org/lang_insert.html it should be combined always with `OR`?

Reply:
`REPLACE INTO table ...` is the same as `INSERT OR REPLACE INTO table ...`. 

> For compatibility with MySQL, the parser allows the use of the single keyword REPLACE as an alias for "INSERT OR REPLACE".